azurerm/internal/services/apimanagement/api_management_custom_domain_resource.go

```diff
@@ -189,6 +189,7 @@ func apiManagementCustomDomainRead(d *schema.ResourceData, meta interface{}) err
 		configs := flattenApiManagementHostnameConfiguration(resp.ServiceProperties.HostnameConfigurations, d)
 		for _, config := range configs {
 			for key, v := range config.(map[string]interface{}) {
+				// nolint R001: ResourceData.Set() key argument should be string literal
```

Comment:
Could you revert this one please? It's in master on a service unrelated to this PR, so should be fixed separately.

Reply:
I am fine with that, but it was causing a lint error for this PR. If you can grant me an exception for passing all lint tests for the PR I would be happy to do so. 🙂

Reply:
I have just opened PR #9298 to fix this issue as suggested above. Thanks!